README.md

```
@@ -187,12 +187,12 @@ WhatsApp provides a multi-device API that allows Baileys to be authenticated as
 > **[Here](#example-to-start) is a simple example of event handling**
 
 > [!TIP]
-> **You can see all supported socket configs [here](https://baileys.whiskeysockets.io/types/SocketConfig.html) (Recommended)**
+> **You can see all supported socket configs [here](https://baileys.wiki/docs/api/type-aliases/SocketConfig/) (Recommended)**
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)